assets/cms/js/vue/Manager.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable no-new */
+/* global Vue */
+import Context from './Context'
+
+class Manager {
+    /**
+     * Ensures that our Concrete.Vue manager is available on the window object.
+     * Note: Do NOT call this before the global Concrete object is created in the CMS context.
+     *
+     * @param window
+     */
+    static bindToWindow(window) {
+        if (!window.Concrete) {
+            window.Concrete = {}
+        }
+        if (!window.Concrete.Vue) {
+            window.Concrete.Vue = new Manager()
+            window.Concrete.Vue.contexts = []
+        }
+    }
+
+    /**
+     * Returns the Vue Context object for the current string `context`
+     *
+     * @param context
+     * @returns {*}
+     */
+    getContext(context) {
+        return window.Concrete.Vue.contexts.filter(obj => obj.context === context)[0]
```

Comment:
If window.Concrete.Vue.contexts becomes an object you can just access this by key. It should probably return a default empty component vue context though